### PR TITLE
Added UI test for an invalid activaiton key copy

### PR DESCRIPTION
**Pull request by JacobCallahan**
I also added a base activation key to reduce extra work, cut test
time from both copy tests by ~120 seconds.

Results
-------
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok
@Test: Create Activation key and fail copying it ... ok

Ran 7 tests in 188.598s

### CODE REVIEW

File: tests/foreman/ui/test_activationkey.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from robottelo.common.helpers import (
     valid_names_list, invalid_names_list,
     valid_data_list, get_server_credentials)
-from robottelo.ui.factory import make_activationkey
+from robottelo.ui.factory import make_activationkey, set_context
```
Comment (JacobCallahan) [inline tests/foreman/ui/test_activationkey.py]: had to include a set context to select the class' organization.

File: tests/foreman/ui/test_activationkey.py [hunk 2]
```diff
@@ -50,6 +50,8 @@ def setUpClass(cls):
         cls.org_label = org_attrs['label']
         cls.org_name = org_attrs['name']
         cls.org_id = org_attrs['id']
+        cls.base_key_name = entities.ActivationKey(
+            organization=cls.org_id).create()['name']
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_activationkey.py]: `s/create/create_json/`

File: tests/foreman/ui/test_activationkey.py [hunk 3]
```diff
@@ -1388,10 +1390,34 @@ def test_positive_copy_activation_key(self, new_name):
...
+        @Feature: Activation key -Negative Copy
+
+        @Steps:
+        1. Create Activation key
+        2. Copy the Activation key with an invalid name
+
+        @Assert: Activation Key copy does not exist
+
+        """
+
```
Comment (Ichimonji10) [inline tests/foreman/ui/test_activationkey.py]: Drop this line?

**Comment by Ichimonji10** [comment]
Triggered Travis rebuild.

File: tests/foreman/ui/test_activationkey.py (hunk 3 above)
Comment (JacobCallahan) [inline tests/foreman/ui/test_activationkey.py]: i'll do that for both. The rest of the test file needs some work as well, but i'll leave that for another day.

File: tests/foreman/ui/test_activationkey.py (hunk 2 above)
Comment (JacobCallahan) [inline tests/foreman/ui/test_activationkey.py]: oops, copied the old way.

Comment (Ichimonji10) [inline tests/foreman/ui/test_activationkey.py]: Thank you. This is a subtle thing that has no ill effect right now, so it's easy to miss. :)

**Comment by JacobCallahan** [comment]
@Ichimonji10 comments addressed in latest version.

**Comment by Ichimonji10** [comment]
ACK

Thank you, @JacobCallahan.

**Comment by elyezer** [comment]
ACK

**Pull request merged**